File: output/gemma4-e2b/avatar/Java/Python/atcoder_ABC143_A.py
import sys

try:
    # Read all input and split by whitespace to get the numbers
    data = sys.stdin.read().split()
    
    if len(data) >= 2:
        a = int(data[0])
        b = int(data[1])
        
        # Calculate c = a - b * 2
        c = a - b * 2
        
        # Ensure c is not negative
        if c < 0:
            c = 0
            
        # Print the result
        print(c)
        
except Exception:
    # Suppress exceptions as in the original Java code
    pass

File: output/gemma4-e2b/avatar/Python/Java/atcoder_ABC143_A.java
import java.util.Scanner;

public class atcoder_ABC143_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // Read the input line containing A and B
        if (scanner.hasNextLine()) {
            String line = scanner.nextLine();
            // Split the line by whitespace and parse to integers
            String[] parts = line.split("\\s+");
            
            if (parts.length >= 2) {
                int A = Integer.parseInt(parts[0]);
                int B = Integer.parseInt(parts[1]);
                
                // Calculate ans = A - 2 * B
                int ans = A - 2 * B;
                
                // if ans < 0: ans = 0
                if (ans < 0) {
                    ans = 0;
                }
                
                // print ans
                System.out.println(ans);
            }
        }
        
        scanner.close();
    }
}